docs/lmsIntegrationInstructor.md

```diff
@@ -36,32 +36,30 @@ take them into your course instance.
 
 Canvas courses must be "Published" for certain integration functionality, like grade reporting, to work.
 
-## "LTI 1.3" tab in your PrairieLearn course instance
+## Integrations tab in your PrairieLearn course instance
 
-Once a Canvas course is linked with your PrairieLearn course instance, you will see an
-"LTI 1.3" tab in the PrairieLearn course instance instructor view. All of the Canvas
-integration features are managed from that page.
+Use the course instance instructor Integrations tab to manage all integration configurations.
```

Comment:
```suggestion
Use the Integrations tab in your course instance to manage all integration configurations.
```

Reply:
That's much clearer, thanks. Updated in 11fd31e589